webknossos/webknossos/client/_upload_dataset.py

```diff
@@ -88,6 +91,14 @@ def upload_dataset(
     simultaneous_uploads = jobs if jobs is not None else DEFAULT_SIMULTANEOUS_UPLOADS
     if "PYTEST_CURRENT_TEST" in os.environ:
         simultaneous_uploads = 1
+    response = new_dataset_name_is_valid.sync_detailed(
+        organization_name=context.organization,
+        data_set_name=new_dataset_name,
+        client=context.generated_auth_client,
+    )
+    assert (
+        response.status_code == 200
+    ), f"Dataset name {context.organization}/{new_dataset_name} does not seem to be valid: {response}"
```

Comment:
It seem's weird that it's not necessary to update the snapshots, since the upload is also tested. Probably some bug related to the testing, I'll check it when working on #514

Reply:
Nevermind, the tests are actually failing due to the tests :see_no_evil: I'll commit an update for those

Reply:
@jstriebel ping for cassette update

Reply:
It was a bit more complex, please double-check my changes in commit 93f7398. I removed the cloning of the wk-repo and used a docker-compose file instead, where the databases are not persisted anymore. Also, I added some filters for vcr.py, so that the timestamps in some requests are ignored. More cleanup is coming with #514